src/mlpack/tests/cv_test.cpp

```diff
@@ -73,6 +74,25 @@ BOOST_AUTO_TEST_CASE(BinaryClassificationMetricsTest)
   BOOST_REQUIRE_CLOSE(F1<Binary>::Evaluate(lr, data, labels), f1, 1e-5);
 }
 
+/**
+ * Test for confusion matrix.
+ */
+BOOST_AUTO_TEST_CASE(ConfusionMatrixTest)
+{
+  // Labels that will be considered as "ground truth".
+  arma::Row<size_t> labels("0 0 1 0 0  1 0 1 0 1");
+
+  // predicted labels.
```

Comment:
This should start with a capital letter.

Reply:
@zoq  , made appropriate changes